README.md

```diff
@@ -97,22 +104,29 @@ create virtual machines for hosting the containers.
 
 ##### Kernel
 
-The [`linux`](https://github.com/kata-containers/linux) kernel is used by the hypervisor to boot the guest image.
+The [Linux\*](https://github.com/kata-containers/linux) kernel is used by the hypervisor to boot the guest image.
 
 ### Documentation
 
-The [`documentation`](https://github.com/kata-containers/documentation)
+The [documentation](https://github.com/kata-containers/documentation)
 repository hosts documentation common to all code components.
 
+### Packaging
+
+The [packaging](https://github.com/kata-containers/packaging) repository is
```

Comment:
Passive voice. Consider:
We use the [packaging repository](https://github.com/kata-containers/packaging) to create packages for the [system components](#kata-containers-developed-components) including [rootfs](#os-builder) and the [kernel](#kernel) images.